### ISSUE TITLE
New CLI => Need to update the 'daemon' page

### ISSUE THREAD

**Issue by grctest** [gridcoin-community/Gridcoin-Site#94]
We have a 'daemon' page showing all the possible commands in the Gridcoin client & the CLI has been uplifted which will require changes to this command page. The wiki may need updating too, but out of scope of this repo.

**Comment by barton2526** [comment]
Need help from @Foggyx420 for this 😦

**Comment by Foggyx420** [comment]
Ok will compile here in abit

**Comment by barton2526** [comment]
This can be closed once PR #115 is merged
